hw03_smithj7.py

```diff
@@ -0,0 +1,103 @@
+
+
+
+#################################################################################
+# Author: Juno Smith
+# Username: smithj
+#
+# Assignment: Hw03
+# Purpose: HW
+# Google Doc Link:https://docs.google.com/document/d/12MjBFSdEOX1sSblb_gTghHuoPa7JGxCLgrh2dcQSNbs/edit
+#
+#################################################################################
+# Acknowledgements:
+# Happy meal
+#
+#################################################################################
+import turtle
+turtle.colormode(255)
+
+
+def drawsquare(JK):     ##Draws House
+    """
+    Drawing the Square Turtle
+    """
+    JK.color('red')
+    JK.pensize(20)
+    JK.penup()
+    JK.goto(-300,-300)
+    JK.pendown()
+
+    for i in range(4):
+        JK.forward(250)
+        JK.left(90)
+
+    # ...
+
+
+def drawTriangle(JK):       ##Draws roof
+    JK.pencolor(255,199,44)
+    JK.pensize(25)
+    JK.penup()
+    JK.goto(-335,-55)
+    JK.pendown()
+
+
```

Comment:
Extra lines are fine to delineate mental chunks. Keep it to a single line though.